cmd/aznum2words-webapp/router/router.go

```diff
@@ -0,0 +1,25 @@
+package router
+
+import (
+	"github.com/labstack/echo/v4"
+	echomiddleware "github.com/labstack/echo/v4/middleware"
+)
+
+var LoggerConfig = echomiddleware.LoggerConfig{
```

Comment:
**LoggerConfig** was being declared as exported variable so that other code routines may access and change it unintentionally

If it is not expected thing, please correct it.